packages/ngtools/webpack/src/ngcc_processor.ts

```diff
@@ -54,6 +55,46 @@ export class NgccProcessor {
       return;
     }
 
+    // Perform a ngcc run check to determine if an initial execution is required.
+    // If a run hash file exists that matches the current package manager lock file and the
+    // project's tsconfig, then an initial ngcc run has already been performed.
+    let skipProcessing = false;
+    let runHashFilePath: string | undefined;
+    const ngccPath = path.join(this._nodeModulesDirectory, '.ngcc');
```

Comment:
Could you make this file more CLI specific? `.ngcc` feels a bit like it is stepping on someone else's namespace :-)

Reply:
It’s actually a folder 😜